### DISCUSSION TITLE
Excellent Battery Life With 1.13.0

### DISCUSSION THREAD

**Discussion by Sn3akyP3t3** [InfiniTimeOrg/InfiniTime#1835] [General]
I just wanted to drop some general praise to those involved in the changes that improved battery life immensely!  Prior to updating to 1.13.0 my battery life ranged from around 3 to 4 days.  Now I'm seeing battery life pushing beyond a week and still going.  I don't know exactly what changed, but I'm liking what I'm seeing so far and wanted to drop some praise for this!

**Comment by JF002** [comment]
Thanks for the feedback, much appreciated :heart: 

We indeed worked on a few optimizations : https://github.com/InfiniTimeOrg/InfiniTime/pull/1758, https://github.com/InfiniTimeOrg/InfiniTime/pull/1756 and https://github.com/InfiniTimeOrg/InfiniTime/pull/1755. 

Basically, those changes consists in powering off most of the MCU when it's going to sleep. This ensure the lowest power usage.

If you want more details, we started working on those changes when @aykevl [shared their findings regarding the power usage of the PineTime](https://github.com/InfiniTimeOrg/InfiniTime/issues/53#issuecomment-1511657606).

**Comment by bbadrignans** [comment]
Yes, many thanks for this big improvement !
On my side here is some results :
- Prior update : up to 10 battery days from 100% to 0%
- After update : up to 25 battery days !!

Some information :
- During these 25 days, I didn't wear the watch every day
- I don't use "raise to wake" or any auto wake mechanism, I push the button to wakeup the watch

![image](https://github.com/InfiniTimeOrg/InfiniTime/assets/534410/c87c8a8a-6108-4016-9dbb-f6c6b44752c7)
